adapters/undertone/undertone.go

```diff
@@ -0,0 +1,209 @@
+package undertone
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strconv"
+)
+
+const adapterId = 4
+const adapterVersion = "1.0.0"
+
+type adapter struct {
+	endpoint string
+}
+
+type BidRequestExt struct {
+	Prebid *openrtb_ext.ExtRequestPrebid `json:"prebid"`
+}
+
+type undertoneBidderParams struct {
```

Comment:
`undertoneBidderParams` is defined in `undertone` package. So we can drop `undertone` prefix from `undertoneBidderParams`

```

type bidderParams struct {
	Id      int    `json:"id"`
	Version string `json:"version"`
}
```